Feat: add edit profile page

fatima-katrib opened the pull request:
Resolve #40

fatima-katrib committed:
Feat: add edit profile page

adlogi commented on src/pages/EditProfile/EditProfile.jsx:
```diff
@@ -0,0 +1,136 @@
+// import proto from '../assets/proto.png';
+
+function EditProfile() {
+    const [fullName, setFullName] = useState('');
```
I think we need separate fields/states for firstName and lastName instead of one fullName. You need also states to handle the profile and id images.

adlogi commented:
@fatima-katrib any updates on this?

fatima-katrib commented:
> @fatima-katrib any updates on this?

updated in a new branch you merged today. i removed the profile pic, handled the state of the id and did small changes in the design
